src/UI/Component/Image/Image.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+/* Copyright (c) 2016 Timon Amstutz <timon.amstutz@ilub.unibe.ch> Extended GPL, see docs/LICENSE */
+
+namespace ILIAS\UI\Component\Image;
+
+/**
+ * This describes how a glyph could be modified during construction of UI.
+ *
+ * Interface Image
+ * @package ILIAS\UI\Component\Image
+ */
+interface Image extends \ILIAS\UI\Component\Component {
+	/**
+	 * Types of images
+	 */
+	const STANDARD = "standard";
+	const RESPONSIVE = "responsive";
```

Comment:
This matches the implementations for glyphs. I think it is the right approach for the moment. The naming is directly adapted from the bootstrap equivalents and they really just differ in class (css). If further adaptions are needed they will probably be on a JS or CSS level. The split can be made anytime if the need really arises, since the implementation is hidden for the consumer.